test/quantum/__snapshots__/index-64.test.jsx.snap

```diff
@@ -133,6 +133,32 @@ exports[`renders correctly 1`] = `
           />
         </svg>
       </a>
+      <a
+        href="/playback"
+        title="Playback"
+      >
+        <svg
```

Comment:
What are you using the `svg` and `path` for here?

Reply:
They also seem to be nearly identical except for the className, so it could go in a component.

Reply:
These `*.test.jsx.snap` are snapshots which are used to compare to future tests.  In this case the snapshot has been is updated to show the playback link.